misd/phone-number-bundle/1.3/post-install.txt

```diff
@@ -1,3 +1,9 @@
+<bg=yellow></>
+<bg=yellow> This bundle is Abandoned: use odolbeau/phone-number-bundle instead </>
+
+  composer remove misd/phone-number-bundle --no-script
```

Comment:
Just a reference here if this does get merged, the option should be `--no-scripts`.

Reply:
fixed